instrumentation/github.com/gin-gonic/gin/otelgin/test/gin_test.go

```diff
@@ -169,7 +169,45 @@ func TestTrace200(t *testing.T) {
 	assert.Contains(t, attr, attribute.String("net.host.name", "foobar"))
 	assert.Contains(t, attr, attribute.Int("http.status_code", http.StatusOK))
 	assert.Contains(t, attr, attribute.String("http.method", "GET"))
-	assert.Contains(t, attr, attribute.String("http.route", "/user/:id"))
```

Comment:
This assertions shouldn't be removed. We want to rely on `req.Pattern` rather than use the value ourselves. We don't want to remove the behavior.